packages/completer/src/model.ts

```diff
@@ -390,7 +390,7 @@ export class CompleterModel implements Completer.IModel {
         index > -1
           ? originalItem.label.substring(0, index)
           : originalItem.label;
-      const match = StringExt.matchSumOfSquares(escapeHTML(text), query);
+      const match = StringExt.matchSumOfSquares(escapeHTML(text.toLowerCase()), query.toLowerCase());
```

Comment:
I think using `toLocaleLowerCase()` instead of `toLowerCase()` might be a better call here. It tends to handle language-specific rules more gracefully, which could save us some headaches down the line. Plus, it aligns better with user expectations, especially in multilingual environments. Cheers!